libraries/exoplayer/src/main/java/androidx/media3/exoplayer/offline/SegmentDownloader.java

```diff
@@ -107,7 +108,29 @@ public int compareTo(Segment other) {
    * @param executor An {@link Executor} used to make requests for the media being downloaded.
    *     Providing an {@link Executor} that uses multiple threads will speed up the download by
    *     allowing parts of it to be executed in parallel.
+   * @param maxMergedSegmentStartTimeDiffUs max merged size for each segment duration
```

Comment:
nit: Let's be a bit more verbose here for clarity. This will be probably reviewed/changed again so fine to leave. I'd say something like

@param maxMergedSegmentStartTimeDiffUs The maximum difference of the start time of two segments, up to which the segments (of the same URI) should be merged into a single download segment, in milliseconds.